.vscode/launch.json

```diff
@@ -30,7 +31,7 @@
 			"outFiles": ["${workspaceFolder}/out/**/*.js"],
 			"smartStep": true,
 			"autoAttachChildProcesses": true,
-			"skipFiles": ["<node_internals>/**","${workspaceFolder}/node_modules/tslib/**"],
+			"skipFiles": ["<node_internals>/**","${workspaceFolder}/node_modules/tslib/**", "C:/users/rbuckton/AppData/Local/Programs/**"],
```

Comment:
Is this leftover?

Reply:
Ah, yeah. Debugging and stepping over vscode internals since they weren't excluded.